### PR TITLE
Feature/ummg support sa dif10

### PR DESCRIPTION
**Pull request by spa0002**
Added check for organization_short_name_gcmd_check in checks.json, dif fields in rule_mapping

organization_short_long_name_consistency_check
- added to rule_mapping, checks, check messages

Updated "characteristic_data_type"
- changed "rule_name" to more generic rule name

"- Updated field names in rule_mapping.json
- Removed ""dependencies"" from checks.json
- Moved ""dependencies"" to rule_mapping.json
- Edited validate_datetime_against_granules and validate_ending_datetime_against_granules in datetime_validator.py"

### CODE REVIEW

File: pyQuARC/code/datetime_validator.py [hunk 1]
```diff
@@ -89,7 +89,7 @@ def validate_datetime_against_granules(datetime, collection_shortname, sort_key,
         Returns:
             (dict) An object with the validity of the check and the instance
         """
-        granules = requests.get(f'{CMR_URL}/search/granules.json?short_name={collection_shortname}&sort_key[]=-{sort_key}').json()
+        granules = requests.get(f'{CMR_URL}/search/granules.json?short_name={collection_shortname}&sort_key[]={sort_key}').json()
```
Comment (slesaad) [inline pyQuARC/code/datetime_validator.py:92]: why this change? it should basically do the same thing?

File: pyQuARC/schemas/checks.json [hunk 2]
```diff
@@ -222,21 +232,11 @@
...
         "available": true
     },
     "validate_beginning_datetime_against_granules": {
         "data_type": "datetime",
         "check_function": "validate_beginning_datetime_against_granules",
-        "dependencies": [
-            [
-                "datetime_format_check"
-            ]
-        ],
```
Comment (slesaad) [inline pyQuARC/schemas/checks.json:239]: why ?

**Review by svbagwell — APPROVED** [review]

File: pyQuARC/code/datetime_validator.py (hunk 1 above)
Comment (spa0002) [reply pyQuARC/code/datetime_validator.py:92]: Since the validate_beginning_datetime_against_granules function uses validate_datetime_against_granules function. With the "-" before the sort_key (in this case, time_start), it compares the collection's beginning_datetime to the beginning datetime of the last granule.
